package.json

```diff
@@ -59,6 +59,10 @@
   "bugs": {
     "url": "https://github.com/brave/brave-browser/issues"
   },
+  "engines": {
+    "node": ">=20.0.0 <22.0.0",
+    "npm": ">=10.0.0"
+  },
```

Comment:
сс @petemill @fallaciousreasoning ptal on version range restrictions I'm adding here. looks good?

`node` can check its own version (at least v20+), but `npm` can't without adding `engine-strict=true` to `.npmrc`.
I don't think we need `engine-strict=true`, because our setup works fine with `yarn`, but for CI and other places where we use `npm` this PR will validate `npm` version as a part of most popular `npm run ...` commands.

Reply:
Version ranges seem fine to me - I guess we'll need to work out why v22 doesn't work eventually but it isn't urgent